doc/nrf/releases/release-notes-latest.rst

```diff
@@ -105,6 +154,23 @@ The following list summarizes the most important changes inherited from upstream
     Use :cpp:func:`bt_le_adv_start` instead, with :cpp:member:`bt_le_adv_param::peer` set to the remote peer's address.
   * Deprecated the ``BT_LE_ADV_*`` macros.
     Use the ``BT_GAP_ADV_*`` enums instead.
+  * L2CAP RX MTU is now controlled by CONFIG_BT_L2CAP_RX_MTU when
+    CONFIG_BT_ACL_FLOW_CONTROL is disabled, previously this was controlled
+    by CONFIG_BT_RX_BUF_LEN. If CONFIG_BT_RX_BUF_LEN has been changed from its
+    default value then CONFIG_BT_L2CAP_RX_MTU should be set to
+    CONFIG_BT_RX_BUF_LEN - 8.
+  * host: Added support for periodic advertisement.
+  * host: Added bond_deleted callback.
+  * host: Refactor the BR/EDR SSP feature out into it's own source file.
+  * host: Allow Start persistent advertiser with maximum connection number reached.
+  * host: Fixed settings of Advertisng Data on extended advertising instances.
+  * host: SMP: Reject legacy pairing early in SC only mode.
+  * Fixed infinite recursion possibility in persistent advertising.
+  * GATT: Fixed issue with bt_gatt_service_unregister not clearing CCC information, which might cause no space to stor ccc configuration.
+  * L2CAP: Elevate the security level before sending the connection request if the application has set a required security level on the channel.
+  * GATT: Added option to disable security checks, see :option: `CONFIG_BT_CONN_DISABLE_SECURITY`.
+  * GATT: Automatic discover of CCC when subscribing, see :option:`CONFIG_BT_GATT_AUTO_DISCOVER_CCC`.
+  * GATT: Fixed the issue where peripheral may not store CCC in non-volatile memory in case of multiple CCC change (due to race condition).
 
```

Comment:
host: Fixed deadlock in receiving disconnected event when disconnecting with pending GATT Write Commands
host: Fixed issue where persistent advertiser would not be started due to  race-condition

Reply:
added